node/p2p/p2p.go

```diff
@@ -0,0 +1,15 @@
+package p2p
+
+import (
+	"go.uber.org/fx"
+
+	"github.com/celestiaorg/celestia-node/node/config"
+)
+
+func P2P(cfg *config.Config) fx.Option {
```

Comment:
Should we call this `P2PComponents` ? Same way as we are calling `fullComponents` and `lightComponents`?